src/librbd/ImageCtx.cc

```diff
@@ -67,7 +67,11 @@ namespace librbd {
       object_cacher(NULL), writeback_handler(NULL), object_set(NULL),
       readahead(),
       total_bytes_read(0), copyup_finisher(NULL),
-      object_map(*this)
+      object_map(*this),
+      thread_pool(cct, "librbd::thread_pool", cct->_conf->rbd_op_threads,
+                  "rbd_op_threads"),
```

Comment:
Sounds good to me -- assuming you can't think of a good use case for different thread counts per image via the new metadata tool.